examples/notebooks/use-cases/prod-forge-nexus.yml

```diff
@@ -32,6 +32,15 @@ Resolvers:
     - resolver: OntologyResolver
       origin: store
       source: BlueBrainNexus
+      contexts: 
+        - name: celltypes
```

Comment:
Shall we further restrict "celltypes" to m- or e-type?
In case of two case-insensitively identical labels between m- and e-types.

Reply:
This doesn't require change of implementation, you can give whatever you want in the configuration file

Reply:
something like:
`contexts: 
        - name: mtypes          
          property: rdfs:subClassOf*
          position: object
          value: http://bbp.epfl.ch/neurosciencegraph/ontologies/core/bmo/NeuronMorphologicalType
`